spec/robots/accession/end_accession_spec.rb

```diff
@@ -24,9 +24,10 @@
   let(:apo_id) { 'druid:mx121xx1234' }
   let(:process) { instance_double(Dor::Workflow::Response::Process, lane_id: 'default') }
   let(:workflow_client) { instance_double(Dor::Workflow::Client, create_workflow_by_name: nil, process: process) }
```

Comment:
Can `create_workflow_by_name` be removed here?

Reply:
No, see L60.